core/primitives/src/snapshots/near_primitives__views__tests__runtime_config_view.snap

```diff
@@ -3,7 +3,7 @@ source: core/primitives/src/views.rs
 expression: "&view"
```

Comment:
This has changed due to the relevant test now testing the actual configuration, rather than the `::test()` one. I believe snapshotting the true configuration is exactly the point of the test, as it is intended to discover changes to the production parameters, and using a snapshot of `::test()` naturally does not actually test for that.